system/security/securesocket/securesocket.cpp

```diff
@@ -666,8 +666,12 @@ int CSecureSocket::secure_accept(int logLevel)
     else if(err < 0)
     {
         int ret = SSL_get_error(m_ssl, err);
+        unsigned long errnum = ERR_get_error();
+        // See note above, SSL_accept() might also return -1
+        if ( (logLevel < SSLogNormal) && (ret == SSL_ERROR_SYSCALL) && (errnum == 0) )
```

Comment:
question: should this be <= SSLogNormal ?

Reply:
> && (errnum == 0)

If this is intented (return only when ERR_get_error returns no error?), it could with a comment,
or wa it supposed to be || (errnum == 0) ?

Reply:
Yes, its exactly like the above case, except when err == -1 we can also get errnum and in this particular situation it is zero.
So its either we add the additional errnum == 0 check or we skip it, its not meant to be an || errnum == 0.
Like above I thought SSLogNormal would still log, but its my opinion.  We could make this and above be only if > SSLogNormal.

Reply:
ok, it's only that it's non obvious from reading the code (ERR_get_error() from man implies if 0 there is no error).
Perhaps add a small comment to clarify for future readability.

Reply:
changed check and added comment.